packages/core/src/search/moon-components.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@heathmont/moon-components' {
+  export const Button: any;
+}
```

Comment:
Fix TS properly

Reply:
I've done all changes except for the placeholder and query prop from string to JSX.Element, because they would give error